src/Request.php

```diff
@@ -301,6 +355,35 @@ public function getHeader(string $key, string $default = ''): string
         return (isset($headers[$key])) ? $headers[$key] : $default;
     }
 
+    /**
+     * Set header
+     *
+     * Method for setting HTTP header parameters.
+     *
+     * @param string $key
+     * @param string $value
+     * @return void
+     */
+    public function setHeader(string $key, string $value): void
```

Comment:
```suggestion
    public function addHeader(string $key, string $value): void
```

For consistency with the response class.